.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules
+node_modules/*
+dist
+dist/*
```

Comment:
I don't think you need to duplicate these.

Reply:
I'll fix that too.